### PR TITLE
copy needed resources to bin dir when run make build

### PR DESCRIPTION
**Pull request by lunny**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -74,6 +74,10 @@ build: $(BIN)/$(EXECUTABLE)
 
 $(BIN)/$(EXECUTABLE): $(wildcard *.go)
 	go build -v -tags '$(TAGS)' -ldflags '-s -w $(LDFLAGS)' -o $@
+	cp -r ./templates ./bin/
+	cp -r ./public ./bin/
+	cp -r ./conf ./bin/
+	cp -r ./scripts ./bin/
```
Comment (tboerger) [inline Makefile:80]: You don't need the scripts

File: Makefile [hunk 2]
```diff
@@ -74,6 +74,10 @@ build: $(BIN)/$(EXECUTABLE)
...
```
Comment (tboerger) [inline Makefile:81]: Maybe we should build this one task simply into the root folder?

File: Makefile (hunk 1 above)
Comment (lunny) [reply Makefile:80]: why not?

File: Makefile (hunk 2 above)
Comment (joubertredrat) [reply Makefile:81]: @tboerger or build on bin and make a symbolic link on root folder. I will test here to see if works.

Comment (joubertredrat) [reply Makefile:81]: @tboerger works fine with symbolic link.

``` bash
dev@devel01:~/public_html/www/gitea-mine$ ln -s bin/gitea .
dev@devel01:~/public_html/www/gitea-mine$ ls -lah gitea
lrwxrwxrwx 1 dev dev 9 Nov 11 08:45 gitea -> bin/gitea
dev@devel01:~/public_html/www/gitea-mine$ ./gitea web
2016/11/11 08:46:43 [T] Custom path: /home/dev/public_html/www/gitea-mine/custom
2016/11/11 08:46:43 [T] Log path: /home/dev/public_html/www/gitea-mine/log
2016/11/11 08:46:43 [I] Gitea: Git with a cup of tea 0.9.99.0915
2016/11/11 08:46:43 [I] Build Time: 2016-11-11 10:45:11 UTC
2016/11/11 08:46:43 [I] Build Git Hash: 85b8b7f
2016/11/11 08:46:43 [I] Log Mode: File(Info)
2016/11/11 08:46:43 [I] Cache Service Enabled
2016/11/11 08:46:43 [I] Session Service Enabled
2016/11/11 08:46:43 [I] Git Version: 1.9.1
2016/11/11 08:46:43 [I] Run Mode: Production
2016/11/11 08:46:43 [T] Doing: CheckRepoStats
2016/11/11 08:46:43 [I] Listen: http://0.0.0.0:3000
```

**Comment by strk** [comment]
Note that you can pass data directory via the `GITEA_WORK_DIR` env variable (would be saner than copying data into a `bin` directory)

**Comment by tboerger** [comment]
I think we should just target it to the root folder instead of copying or linking.

**Comment by lunny** [comment]
So I close this, and please send a PR to change the target from `bin` to root folder. @tboerger

**Pull request closed without merging**